_rules/html-xml-lang-match-5b7ae0.md

```diff
@@ -19,11 +19,13 @@ authors:
 
 ## Applicability
 
-The root element of the page, if it is an `html` element with both `lang` and `xml:lang` attributes that has a [valid language subtag](#valid-language-subtag).
+The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element with both `lang` and `xml:lang` attributes that has a [valid language subtag](#valid-language-subtag).
```

Comment:
that has a [valid language subtag](#valid-language-subtag) > that have [valid language subtags](#valid-language-subtag)